python/cugraph-pyg/pyproject.toml

```diff
@@ -3,9 +3,11 @@
 [build-system]
 
 requires = [
+    "rapids-build-backend>=0.3.1,<0.4.0.dev0",
```

Comment:
Starting a threaded conversation so this can be collapsed once it's resolved.

I just pushed some commits that will hopefully allow CI to get further.

`cugraph-dgl` and `cugraph-pyg` on this branch were very far behind the `cugraph` repo. For example, I found that none of the changes from https://github.com/rapidsai/cugraph/pull/4393 (merged into `cugraph` 3+ months ago).

I just copied over the packaging changes because I don't understand the current state of the `cugraph-gnn` repo. There have, for example, been code changes to the libraries themselves here (like #46).

Just calling it out for you @alexbarghi-nv in case I've misunderstood and those packaging changes being missed actually means something else went wrong resolving a merge conflict or something.

Reply:
We've been keeping the code up to date with `cugraph`.  All the PRs I merged for cuGraph-DGL/PyG, and all the PRs for WholeGraph in 24.10 were replicated here.  But the WG team said they are not going to merge any 24.12 PRs until we have CI working, and I am respecting their decision.

Reply:
It's very possible I missed some PRs that updated CI

Reply:
That's helpful context, thank you.